lib/Standalone/ConvertXsmmToFunc.cpp

```diff
@@ -344,9 +345,11 @@ struct ConvertUnaryDispatch : public OpRewritePattern<UnaryDispatchOp> {
   LogicalResult matchAndRewrite(UnaryDispatchOp dispatchOp,
                                 PatternRewriter &rewriter) const override {
     Location loc = dispatchOp.getLoc();
-    std::string kindAsString = "xsmm_unary_dispatch";
+    std::string kindAsString = "xsmm_unary_dispatch_";
+    std::string typeAsString = stringifyEnum(dispatchOp.getDataType()).str();
```

Comment:
stringifyEnum returns a `Stringref`?

Reply:
I am returning a .str() from the stringref.

Reply:
ok